cypress/support/commands.ts

```diff
@@ -31,6 +42,9 @@ declare global {
         postAvatarError?: boolean;
         updatePasswordError?: boolean;
         updateEmailError?: boolean;
+        mockPagination?: Pagination;
+        files?: Paginated<MemberStorageItem>;
```

Comment:
I think the type here should be: 
```suggestion
        files?: MemberStorageItem[];
```
And you build the paginated stuff in the mock response in the server.